doc/source/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.rst

```diff
@@ -0,0 +1,95 @@
+======================================
+2D Flow around a cylinder
+======================================
+
+In this example, a flow is passing across a fixed cylinder. The velocity profile of the flow is simulated. The parameter file used is ``cylinder.prm``. This example can be solved using the ``gls_navier_stokes_2d`` solver.
+
+The following schematic describes the simulation.
+
+.. image:: images/geometry_bc.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry_and_bc
+
+
+* bc = 0 (No slip boundary condition)
+* bc = 1 (u = 1; flow in the x-direction)
+* bc = 2 (Slip boundary condition)
+
+Initial conditions
+------------------
+The initial condition implemented describes the movement of the flow in the x-direction.
```

Comment:
Can you add an explanation as to why you set it to 1 for the initial conditions. This is a steady-state problem so why do you need an initial condition?